08.Third-party-API-implementation.md

```diff
@@ -0,0 +1,12 @@
+# 3rd Party API Implementation
+
+Some contributors are creating and maintaining external libraries that could simplify your life on your project. Those contributors created packages that use Gitter API in different shapes (progamming languages, main goals, ...). You can retrieve a list of the current existing packages below, sorted by languages.
```

Comment:
You can also access the Gitter API through one of the 3rd party libraries that may be more suitable to use in your language, platform, and environment.